### DISCUSSION TITLE
What is the new "core mods" page in instance settings?

### DISCUSSION THREAD

**Discussion by BBaoVanC** [PrismLauncher/PrismLauncher#2684] [Q&A]
I don't see this anywhere in the changelogs, issues, or documentation, and the help button goes to a 404. The "Download mods" button seems to show the exact same selection as the regular Mods tab

<img width="792" alt="image" src="https://github.com/user-attachments/assets/ac3ff29d-e382-4710-b185-3a55c6c189ab">

Is this related to jarmods, Forge coremods, or something else entirely?

**Comment by blryface** [comment]
Some forge mods are core mods iirc that behave different or something

**Reply by BBaoVanC** [reply]
That's what I thought at first, but coremods are normally installed the same way as regular ones. The download mods button shows the same selection, including many mods that have no coremods

**Comment by TayouVR** [comment]
see here: https://forums.minecraftforge.net/topic/62870-what-is-a-coremod-and-what-are-it-advantages-and-disadvantages/

**Reply by BBaoVanC** [reply]
What would be the difference between putting a coremod in the coremods directory versus putting it in the regular mods folder like normal? I have some mod jar's under `mods/` that include coremods, is that different?

**Reply by Ryex** [reply]
A mod that includes a core mod has the machinery to load that core mod built in. a raw coremod can not generally be loaded by a modloader

**Reply by BBaoVanC** [reply]
Seems related: https://github.com/MultiMC/Launcher/issues/3348. They say it was only used before 1.6.1 for loading coremods.

Is it a bug that this tab would show up on a 1.12.2 Forge instance? For some reason I'm not seeing it anymore.
